src/components/map/MapService.js

```diff
@@ -424,6 +424,24 @@ goog.require('ga_urlutils_service');
           }
           return olLayer;
         };
+
+        // Make a GetLegendGraphic request
+        this.getLegend = function(layer) {
+          var defer = $q.defer();
+          var params = layer.getSource().getParams();
+          var html = '<img src="' + gaUrlUtils.append(layer.url,
```

Comment:
How about an `alt` attribute indicate non-availability of GetLegnedGraphic?

`alt=No Legend Available` (IMO, does not need translation)